### PR TITLE
[Injimob 695] For esignet and sunbird Vc's show banner message based on the verification call

### PR DESCRIPTION
**Pull request by PuBHARGAVI**

### CODE REVIEW

File: shared/vcjs/verifyCredential.ts [hunk 1]
```diff
@@ -26,10 +27,17 @@ const ProofPurpose = {
 
 export async function verifyCredential(
   verifiableCredential: VerifiableCredential | Credential,
+  vcMetadata: VCMetadata,
 ): Promise<VerificationResult> {
   try {
     let purpose: PublicKeyProofPurpose | AssertionProofPurpose;
-    switch (verifiableCredential.proof.proofPurpose) {
+    const proof = VCMetadata.fromVC(vcMetadata).isFromOpenId4VCI()
```
Comment (vijay151096) [inline shared/vcjs/verifyCredential.ts:34]: can we not add this check "VCMetadata.fromVC(vcMetadata).isFromOpenId4VCI()" here, lets check it before calling and send only the credential, so verifyCredential Method resposibilty is to just verify and not figuring out where the credential would be.

File: components/VCVerification.tsx [hunk 2]
```diff
@@ -21,10 +21,12 @@ export const VCVerification: React.FC = ({wellknown, isVerified}: any) => {
         {statusIcon}
         <Text
           testID="verificationStatus"
-          color={getTextColor(wellknown, Theme.Colors.Details)}
           style={[
             Theme.Styles.detailsValue,
-            {fontFamily: 'Inter_600SemiBold'},
+            {
+              fontFamily: 'Inter_600SemiBold',
```
Comment (sree96) [inline components/VCVerification.tsx:27]: Lets move this to theme file

**Review by sree96 — CHANGES_REQUESTED** [review]
Plz look at the comments

File: components/VCVerification.tsx (hunk 2 above)
Comment (PuBHARGAVI) [reply components/VCVerification.tsx:27]: Done as part of another PR

File: machines/Issuers/IssuersGuards.ts [hunk 3]
```diff
@@ -3,9 +3,12 @@ import {ErrorMessage, Issuers, OIDCErrors} from '../../shared/openId4VCI/Utils';
 import {isHardwareKeystoreExists} from '../../shared/cryptoutil/cryptoUtil';
 import {BiometricCancellationError} from '../../shared/error/BiometricCancellationError';
 import {NETWORK_REQUEST_FAILED, REQUEST_TIMEOUT} from '../../shared/constants';
+import {VerificationErrorType} from '../../shared/vcjs/verifyCredential';
 
 export const IssuersGuards = () => {
   return {
+    isPendingVerificationError: (_context, event) =>
```
[the remark this answers is not in the document]
Comment (PuBHARGAVI) [reply machines/Issuers/IssuersGuards.ts:10]: Done as part of another pr

File: shared/vcjs/verifyCredential.ts (hunk 1 above)
Comment (PuBHARGAVI) [reply shared/vcjs/verifyCredential.ts:34]: Done as part of another PR

**Pull request closed without merging**